daml-assistant/integration-tests/src/Main.hs

```diff
@@ -367,6 +366,11 @@ damlInstallerName
     | isWindows = "daml.exe"
     | otherwise = "daml"
 
+mvn :: String
+mvn
+    | isWindows = "mvn.cmd"
```

Comment:
I thought if you use `shell`, it will automatically pick `mvn.cmd`? Is that only the case if `mvn` does not exist?

Reply:
You're right, thanks! Now it's just `mvn`